nmostesting/NMOSTesting.py

```diff
@@ -1139,6 +1139,13 @@ def main(args):
     print(" * Testing tool running on 'http://{}:{}'. Version '{}'"
           .format(get_default_ip(), core_app.config['PORT'], TOOL_VERSION))
 
+    # Give an API or client that is already running a chance to use the mock services
+    # before running any test cases
+    if CONFIG.MOCK_SERVICES_WARM_UP_DELAY:
+        print(" * Warm up your engines, testing begins in {} seconds..."

```

Comment:
As per above comment

```suggestion
        print(" * Waiting {}s to allow discovery of mock resources"
```